### ISSUE TITLE
Projucer information missing

### ISSUE THREAD

**Issue by lachose1** [DBraun/TD-JUCE#8]
Hello,
I tried compiling your project and everything runs smooth until I gotta build the project in Visual Studio. I received a lot of errors in regards to a library not being included correctly (for instance juce_audio_formats missing and such) so I noticed that you are actually linking against two folders inside of `thirdparty/` which are `JUCE_6` and `JUCE_5` . I didn't have this code locally so I proceeded to download the code from their latest release here : https://github.com/juce-framework/JUCE/releases/tag/6.0.5 . Almost everything built succesfully except from one error, it didn't seem to find : `JUCE_6\modules\juce_audio_processors\format_types\VST3_SDK\pluginterfaces\vst2.x` which seems to be discontinued since a couple of years when Steinberg dropped their VST2 support. I still managed to find an alternative file online in this specific commit from JUCE themselves a couple of years ago : https://github.com/juce-framework/JUCE/tree/a54535bc317b5c005a8cda5c22a9c8d4c6e0c67e/modules/juce_audio_processors/format_types/VST3_SDK/pluginterfaces

However I get one last error : C1189	#error:  "This project was last saved using an outdated version of the Projucer! Re-save this project with the latest version to fix this error." 

I get it in both TD-JUCE-REVERB and TD-JUCE-VST. Now I figured that I could most likely fix this error by using Projucer and creating the correct project myself which is probably something I assume you did yourself. However, this information and all I just told you seem to be missing from your readme so that we can get kickstarted correctly. Could you help me correct it? Thanks!

**Comment by DBraun** [comment]
Hello, thanks for trying out the project. I'm optimistic that you'll get this working soon. I recommend starting from scratch and cloning the repo with git. This will use git submodules to download JUCE_5 and JUCE_6 to the thirdparty folder. As you figured out, the submodules have been set up so that JUCE_5 contains the old VST SDK that is currently missing from JUCE_6. I intend to periodically update JUCE_6 to get the latest features, whereas JUCE_5 will stay as is (with some bloat) just to have the old SDK around.

So if you start over from scratch, you shouldn't have any issues. Projucer is not necessary at all because we're using CMake.

**Comment by lachose1** [comment]
That's strange as I definitely used git clone from the git bash to initially clone your repository. Right after that I changed to the cloned directory and created the `build` folder with mkdir (following your readme) and then the two other steps which were to change to this directory than run cmake .. which correctly created all the directory structure. Let me try again asap and get back to you.

**Comment by lachose1** [comment]
Ok I just noticed that your repository uses submodules. Basically just adding them via 

`
git submodule init
`

followed by

`
git submodule update
`

and everything compiled! Now to test it :)

**Comment by DBraun** [comment]
Great. I hope it goes well. Please follow up with any issues you encounter.

**Comment by DBraun** [comment]
Closing. Please re-open if there's an issue.
